packages/core/src/data-module/TimeSeriesDataModule.ts

```diff
@@ -26,7 +26,7 @@ import type {
 
 export const DEFAULT_CACHE_SETTINGS = {
   ttlDurationMapping: {
-    [1.2 * MINUTE_IN_MS]: SECOND_IN_MS * 1.5,
+    [0.5 * MINUTE_IN_MS]: SECOND_IN_MS,
```

Comment:
If I am reading this right, this would probably halve the amount of data requested?

Reply:
Yup, this is where the 30 seconds of data that we were seeing in the bug bash is coming from.

Reply:
there should be another interval added [1.2 * MINUTE_IN_MS]: SECOND_IN_MS * 5, to maintain what was there before

Reply:
I chose to stick with the intervals we already had since they are tested, and just reduced the right hand value to one second so we can support 1 second rate